index.html

```diff
@@ -301,17 +273,21 @@ <h3>Skills</h3>
         <!-- Contact me section -->
         <section class="contact-me-section">
             <div class="contact-me-section-container">
-                <h1>Contact me</h1>
-                <p>
-                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
-                </p>
-                <form action="https://formspree.io/f/xdojqlba" method="post" class="form-container">
-                    <input type="text" placeholder="Your name" minlength="3" maxlength="30" required>
-                    <input type="email" placeholder="Your email" required>
-                    <textarea name="message" id="message" cols="30" rows="10" minlength="5" maxlength="500" placeholder="Write your message here" required></textarea>
-                    <input type="submit" value="Get It Touch">
-                </form>
-                <span class="footer-indicator"></span>
+                <div class="contact-me-section-content">
+                    <h1>Contact me</h1>
+                    <p>
+                        If you have an application you are interested in developing, a feature that you need built or a project that needs
+                        coding. I’d love to help with it
+                    </p>
+                    <form action="https://formspree.io/f/xdojqlba" method="post" class="form-container">
+                        <input type="text" placeholder="Your name" minlength="3" maxlength="30" required>
+                        <input type="email" placeholder="Your email" required>
+                        <textarea name="message" id="message" cols="30" rows="10" minlength="5" maxlength="500"
+                            placeholder="Write your message here" required></textarea>
+                        <input type="submit" value="Get It Touch">
+                    </form>
+                    <span class="footer-indicator"></span>
+                </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please remove the `<span class="footer-indicator"></span>` tag because it doesn't provide the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/206740775-d5355168-bc75-4742-a3f9-ed669478747a.png)